### DISCUSSION TITLE
Get more detailed debugging information when scanning helm charts

### DISCUSSION THREAD

**Discussion by JAORMX** [aquasecurity/trivy#4367] [Ideas]
If there's an issue with the helm chart that trivy attempts to scan in a configuration scan, trivy silently ignores it. While it's fine that it doesn't fail, it should at least show some information on the issue when enabling debug logging.

**Comment by simar7** [comment]
Trivy does expect the input to be valid in order to find any misconfigurations in it. Having this as a debug output will be too noisy as there might be many non relevant or invalid files, IMO.
